crates/core/src/client/config.rs

```diff
@@ -128,6 +128,8 @@ impl<T: JsonRpcTransport> JsonRpcClientBuilder<T> {
         Self(JsonRpcClient::new(transport))
     }
 
+    // This clippy lint is false positive, trying to make this function `const` but it doesn't work.
+    #[allow(clippy::missing_const_for_fn)]
```

Comment:
interesting! looking at this builder pattern, it feels overkill, maybe that's why clippy is giving this false positive, 
there's one degree of abstraction that isn't useful